tests/Endpoints/DocumentsTest.php

```diff
@@ -84,6 +84,28 @@ public function testAddDocumentsCsv(): void
         $this->assertCount(20, $response);
     }
 
+    public function testAddDocumentsCsvWithCustomSeparator(): void
+    {
+        $index = $this->client->index('documentCsvWithCustomSeparator');
+
+        $fileCsv = fopen('./tests/datasets/songs-custom-separator.csv', 'r');
+        $documentCsv = fread($fileCsv, filesize('./tests/datasets/songs-custom-separator.csv'));
+        fclose($fileCsv);
```

Comment:
why not just `file_get_contents`?

Reply:
Good idea! I'm not much aware of all the PHP features, so I'm glad you suggested that :)